src/kirby/components/badge/badge.component.tns.scss

```
@@ -0,0 +1,48 @@
+@import "../../scss/native-imports";
+
+.kirby-badge Label { 
```

We are already scoped, so maybe just:
```suggestion
.badge { 
```